### DISCUSSION TITLE
Can Poetry colors be changed?

### DISCUSSION THREAD

**Discussion by albertotb** [python-poetry/poetry#3689] [General] [answered]
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] I have searched the [documentation](https://python-poetry.org/docs/) and believe that my question is not covered.

## Issue
Is it possible to change Poetry output colors? I was thinking it would be nice to change them similarly to how you can change the color of the prompt or the different file types

**Comment by ghost** [comment]
Whilst not a poetry-specific solution, changing your terminal emulator's default colours will change the look of poetry's output. For instance, when using `gnome-terminal` creating a new profile allows you to select a new palette of default colours.

I personally use the [nord](https://github.com/arcticicestudio/nord-gnome-terminal) colourscheme, which creates a new profile with new default/system colours.

**Comment by albertotb** [comment]
I understand, but that will change the colors for all the programs running in that terminal. My question is that if I can do it for poetry only

**Comment by martinda** [comment]
I started a similar discussion on disabling all colors in #8838

**Comment by abn** [comment, marked as answer]
Changing colors is not a feature we are actively looking at. 

You can look at Cleo and Poetry codebases if you'd like to implement something yourself as a proposal. 

No guarantees around how it might be prioritised though.

**Reply by martinda** [reply]
Hi. Thanks for the update. I am not sure how Github issue resolution works, but the email I got said "Closed as resolved", however this is more "Closed as won't fix".

**Reply by abn** [reply]
This is a GitHub Discussion as opposed to an issue. GH kind of internally treats it as an issue. Likely why you got that message. But for the purposes of this discussion, the matter is resolved.
